Helper/SupportFormHelper.php

```diff
@@ -386,8 +386,8 @@ public function getConfigData(): array
             'boletoDeliveryDays' => $boletoDeliveryDays,
             'isPayByLinkEnabled' => $isPayByLinkEnabled ? 'Yes' : 'No',
             'useManualCaptureForPaypal' => $useManualCaptureForPaypal ? 'Yes' : 'No',
-            'captureOnShipmentForOpenInvoice' => $captureOnShipmentForOpenInvoice ? 'Yes' : 'No',
-            'autoCaptureOpenInvoice' => $autoCaptureOpenInvoice ? 'Yes' : 'No',
+            'captureOnShipmentForOpenInvoice' => $captureOnShipmentForOpenInvoice,
+            'autoCaptureOpenInvoice' => $autoCaptureOpenInvoice,
```

Comment:
As a follow-up to previous comment, those values will be identical. You can merge them into one field. After changing the values in this function, you need to update support form templates [here](https://github.com/Adyen/adyen-magento2/tree/develop/view/adminhtml/email) which use those fields.